db/model/comments-model.js

```diff
@@ -0,0 +1,19 @@
+const db = require('../connection');
+const { convertTimestampToDate, createRef, formatComments} = require('../seeds/utils');
+
+
+const selectCommentsByArticleId = (article_id) => {
+    return db.query(`SELECT * FROM comments WHERE article_id = $1 
+    ORDER BY created_at desc;`,  [article_id])
+    .then((result) => {
+        if (result.rows.length === 0) {
+            return Promise.reject({
+                status: 404,
+                msg: 'Resource not found'
```

Comment:
What would happen here if the article_id is in the database but there are no comments yet and is an empty array? Would you want to have a error if the article_id exists and there are no comments yet? You should consider how you would handle this when you are getting the same result for two different outcomes.